### PR TITLE
Add German translations for the labels

### PR DESCRIPTION
**Pull request by Hamza2961**

### CODE REVIEW

File: Resources/Private/Language/de.locallang_be.xlf [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                <source>Number of Links</source>
+                <target>Anzahl der Links</target>
+            </trans-unit>
+            <trans-unit id="flexform.el.paginationEnabled.items.0.label">
+                <source>Enabled</source>
+                <target>Aktiviert</target>
+            </trans-unit>
+            <trans-unit id="flexform.el.profileList.label">
+                <source>List of profiles</source>
+                <target>Liste der Profile</target>
```
Comment (torben-fr) [inline Resources/Private/Language/de.locallang_be.xlf:118]: I would change "der" to "von"

File: Resources/Private/Language/de.locallang_tca.xlf [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+                <source>Organisational Level 3</source>
+                <target>Organisatorische Ebene 3</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_address.columns.street.label">
+                <source>Street</source>
+                <target>Straße</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_address.columns.street_number.label">
+                <source>Street Number</source>
+                <target>Straße Nummer</target>
```
Comment (torben-fr) [inline Resources/Private/Language/de.locallang_tca.xlf:74]: That would be "Hausnummer" in German

File: Resources/Private/Language/de.locallang_tca.xlf [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+                <source>Mr.</source>
+                <target>Herr</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_profile.columns.gender.items.ms">
+                <source>Ms.</source>
+                <target>Frau</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_profile.columns.gender.items.diverse">
+                <source>Diverse</source>
+                <target>divers</target>
```
Comment (torben-fr) [inline Resources/Private/Language/de.locallang_tca.xlf:127]: With a capital "D" please

File: Resources/Private/Language/de.locallang_tca.xlf [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+                <source>Title</source>
+                <target>Titel</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_profile.columns.first_name.label">
+                <source>First Name</source>
+                <target>Vorname</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_profile.columns.middle_name.label">
+                <source>Middle Name</source>
+                <target>Mittlerer Name</target>
```
Comment (torben-fr) [inline Resources/Private/Language/de.locallang_tca.xlf:139]: That would be "Zweitname" in German

File: Resources/Private/Language/de.locallang_tca.xlf [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+                <source>Room</source>
+                <target>Zimmer</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_contract.columns.phone_numbers.label">
+                <source>Phone Numbers</source>
+                <target>Telefon Nummern</target>
+            </trans-unit>
+            <trans-unit id="tx_academicpersons_domain_model_contract.columns.email_addresses.label">
+                <source>Email Addresses</source>
+                <target>E-Mail-Adressen</target>
```
Comment (torben-fr) [inline Resources/Private/Language/de.locallang_tca.xlf:260]: The second "-" is not needed, please remove it

**Review by torben-fr — CHANGES_REQUESTED** [review]

**Review by Riiiad — APPROVED** [review]

**Review by Riiiad — APPROVED** [review]

**Review by torben-fr — APPROVED** [review]

**Pull request merged**